io/plugins/eu.esdihumboldt.hale.io.json/src/eu/esdihumboldt/hale/io/json/writer/NamespaceManager.java

```diff
@@ -13,7 +13,7 @@
  *     wetransform GmbH <http://www.wetransform.to>
  */
 
-package eu.esdihumboldt.hale.io.json.internal;
+package eu.esdihumboldt.hale.io.json.writer;
```

Comment:
Any specific reason for moving this class and many others?

Like this one, many of the classes moved are not only relevant for writing Json but are used for reading Json as well, and were used there before any of the code to write Json was added.